modules/farm_fd2/src/entrypoints/cover_crop_seeding/lib.submit.multiplepasses.unit.cy.js

```diff
@@ -0,0 +1,89 @@
+import { lib } from './lib.js';
+import * as farmosUtil from '@libs/farmosUtil/farmosUtil';
+
+describe('Submission with multiple passes', () => {
+  const form = {
+    date: '1950-01-02',
+    crops: ['BEAN', 'CARROT'],
+    location: 'ALF',
+    beds: ['ALF-1', 'ALF-3'],
+    areaSeeded: 50,
+    seedApplicationEquipment: ['Tractor'],
+    seedIncorporationEquipment: ['Rake'],
+    seedApplicationDepth: 6,
+    seedApplicationSpeed: 5,
+    seedApplicationPasses: 3,
```

Comment:
Let's just do 2 passes unless there is a good reason to do 3.  This will make the test faster.